maps/src/main/java/org/odk/collect/maps/MapFragment.java

```diff
@@ -123,7 +123,7 @@ void addTo(
      * By default, the marker icon will be R.drawable.ic_map_point.
      * Returns a positive integer, the featureId for the newly added shape.
      */
-    int addMarker(MapPoint point, boolean draggable, @IconAnchor String iconAnchor);
+    int addMarker(MapPoint point, boolean draggable, @IconAnchor String iconAnchor, int iconDrawableId);
```

Comment:
Consider overloading instead. `addMarker(point, iconDrawableId, iconAnchor)`, eg. That way you won't end up having to duplicate the resource.

Reply:
I don't know... ideally it was kotlin and I could use defaults.